### PR TITLE
[Test-1] Add test cases for ConfigurationFactory

### LINKED ISSUE #188: Add unit test to fescar-config module
Add unit test to fescar-config module

### PR DESCRIPTION
**Pull request by xingfudeshi**
2.[Test-1] Add test cases for ConfigurationFactory

### Ⅰ. Describe what this PR did
Add test cases for ConfigurationFactory

### Ⅱ. Does this pull request fix one issue?

fixes #188

### CODE REVIEW

File: config/src/test/java/com/alibaba/fescar/config/ConfigurationFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+     */
+    @Test
+    public void testGetInstance() throws NotSupportYetException {
+        if (ConfigType.Nacos.equals(CONFIG_TYPE)) {
+            assertThat(CONFIGURATION_INSTANCE).isInstanceOf(NacosConfiguration.class);
+        } else if (ConfigType.Apollo.equals(CONFIG_TYPE)) {
+            assertThat(CONFIGURATION_INSTANCE).isInstanceOf(ApolloConfiguration.class);
+        } else if (ConfigType.File.equals(CONFIG_TYPE)) {
+            assertThat(CONFIGURATION_INSTANCE).isInstanceOf(FileConfiguration.class);
+        }
```
Comment (slievrly) [inline config/src/test/java/com/alibaba/fescar/config/ConfigurationFactoryTest.java]: something for else?

File: config/src/test/java/com/alibaba/fescar/config/ConfigurationFactoryTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.alibaba.fescar.config;
+
+import com.alibaba.fescar.common.exception.NotSupportYetException;
+import org.junit.Test;
+
+import static com.alibaba.fescar.config.ConfigurationFactory.FILE_INSTANCE;
+import static org.assertj.core.api.Assertions.*;
```
Comment (slievrly) [inline config/src/test/java/com/alibaba/fescar/config/ConfigurationFactoryTest.java]: *？

File: config/src/test/java/com/alibaba/fescar/config/ConfigurationFactoryTest.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+     */
+    @Test
+    public void testGetInstance() {
+        //try to get NotSupportYetException
+        Throwable thrown = catchThrowable(ConfigurationFactory::getInstance);
+
+        if (thrown instanceof NotSupportYetException) {//NotSupportYetException case
+            assertThat(thrown).hasMessage("not support register type:" + CONFIG_TYPE);
+        } else if (ConfigType.Nacos.equals(CONFIG_TYPE)) {//Nacos case
+            Configuration configuration = ConfigurationFactory.getInstance();
```
Comment (slievrly) [inline config/src/test/java/com/alibaba/fescar/config/ConfigurationFactoryTest.java]: Why do you want to get each time？

File: config/src/test/java/com/alibaba/fescar/config/ConfigurationFactoryTest.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            FILE_INSTANCE.getConfig(ConfigurationKeys.FILE_ROOT_CONFIG + ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR
+                    + ConfigurationKeys.FILE_ROOT_TYPE));
+
+    /**
+     * test for getInstance method
+     */
+    @Test
+    public void testGetInstance() {
+        try {
+            Configuration configuration = ConfigurationFactory.getInstance();
```
Comment (github-ygy) [inline config/src/test/java/com/alibaba/fescar/config/ConfigurationFactoryTest.java]: more better to use FILE_INSTANCE putconfig to test appointed the conf？

Comment (xingfudeshi) [reply config/src/test/java/com/alibaba/fescar/config/ConfigurationFactoryTest.java]: thanks for the review,i will improve it.

**Pull request merged**